src/main/java/com/liferay/faces/util/config/WebConfigParam.java

```diff
@@ -17,16 +17,49 @@
 
 import javax.faces.context.ExternalContext;
 
-import com.liferay.faces.util.config.ConfigParam;
-import com.liferay.faces.util.config.WebConfigParamUtil;
+import com.liferay.faces.util.cache.CacheFactory;
 import com.liferay.faces.util.helper.BooleanHelper;
+import com.liferay.faces.util.i18n.I18n;
 
 
 /**
  * @author  Neil Griffin
  */
 public enum WebConfigParam implements ConfigParam<ExternalContext> {
 
+	/**
+	 * Integer indicating the default initial cache capacity for {@link com.liferay.faces.util.cache.Cache} instances
+	 * created by {@link CacheFactory}. The default value of this param is 16. For more details, see {@link
+	 * java.util.HashMap#HashMap()}
+	 *
+	 * @since  1.1
+	 * @since  2.1
+	 * @since  3.1
+	 */
+	DefaultInitialCacheCapacity(CacheFactory.class.getName() + ".defaultInitialCacheCapacity", 16),
```

Comment:
@stiemannkj1 We let developers fine-tune the maximum capacity on an individual cache basis, but we only let them fine-tune the initial capacity for all caches. I wonder if this is an indicator that we should remove the CacheFactory method that omits the `initialCapacity` method argument and always force it to be passed. In other words, if we are always checking a web.xml init-param to pass for `maxCapacity` then why not always do both?